src/components/switch/switch.tsx

```diff
@@ -86,28 +88,44 @@ export class Switch {
 
     public render() {
         return [
-            <div
+            <button
+                id={this.fieldId}
                 class={{
                     'mdc-switch': true,
-                    'mdc-switch--disabled': this.disabled || this.readonly,
                     'lime-switch--readonly': this.readonly,
```

Comment:
The design of Switch has changed a lot in their new version, which basically cancels our customizations. Specially fr the `readonly` mode. We will have to go through this

Reply:
I'd love it if either 1) we think it's ok if the looks change for a little while, so we can merge this and fix the styling later, or 2) we can prioritise fixing the styling so we can get this merged relatively soon.

The above assuming that we don't find any _other_ major stoppers of course. So please continue testing this for changes and bugs, before jumping on fixing the styling for limel-switch 😊

Reply:
@Kiarokh Ping on ☝️

Reply:
@adrianschmidt I've gone through most of the components, and nothing catches my attention as a bug yet. So we can merge this if you want. I'm working on the switch now. If I manage to fix the switch in this PR, I'd push it as a fixup. If you merge it first, I'll add it as a new PR ;)

Reply:
Good stuff! I decided to squash them into `chore: fixes for MDC 13`. Since we don't really announce to the user of lime-elements what version of MDC we're using, I didn't think that fixes that are meant mainly to make the v13 switch look like the v11 switch was something that needed listing in the changelog…